<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MLB Stadium Information</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="buttonLayout">
      <item>
       <widget class="QComboBox" name="teamComboBox">
        <property name="minimumWidth">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="teamInfoButton">
        <property name="text">
         <string>Show Team Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortByTeamButton">
        <property name="text">
         <string>Sort by Team Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortByStadiumButton">
        <property name="text">
         <string>Sort by Stadium Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="americanLeagueButton">
        <property name="text">
         <string>American League Teams</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nationalLeagueButton">
        <property name="text">
         <string>National League Teams</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="typologyButton">
        <property name="text">
         <string>Sort by Park Typology</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openRoofButton">
        <property name="text">
         <string>Teams with Open Roof</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dateOpenedButton">
        <property name="text">
         <string>Sort by Date Opened</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="capacityButton">
        <property name="text">
         <string>Sort by Capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="maxCenterFieldButton">
        <property name="text">
         <string>Greatest Center Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minCenterFieldButton">
        <property name="text">
         <string>Smallest Center Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importCSVButton">
        <property name="text">
         <string>Import CSV Data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="resultsTable">
      <property name="minimumWidth">
       <number>700</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui> 